tests/unit/preprocessor/_mask/test_mask.py

```diff
@@ -91,6 +96,13 @@ def test_get_fx_mask(self):
         expected = np.array([True, False, True])
         assert_array_equal(expected, computed)
 
+    def test_mask_glaciated(self):
+        """Test to mask glaciated (NE mask)"""
+        result = mask_glaciated(self.arr, mask_out='glaciated')
+        expected = np.ma.masked_array(self.data2, mask=\
+                                      np.array([[True, True], [False, False]]))
+        _assert_masked_array_equal(result.data, expected)
+
```

Comment:
add the check on `.mask` too, you found the bug and you not fixing it here :grin:

Reply:
I did ;-) I even wrote a separate function for it! :beer:

Reply:
yes, I am officially blind :man_facepalming: